### PR TITLE
Update CI from plugin-template

### PR DESCRIPTION
**Pull request by fao89**
[noissue]

### CODE REVIEW

File: template_config.yml [hunk 1]
```diff
@@ -48,9 +48,8 @@ redmine_project: ansible_plugin
 stable_branch: null
 test_bindings: false
 test_cli: true
-test_fips_nightly: false
 test_performance: false
-test_released_plugin_with_next_pulpcore_release: true
+test_released_plugin_with_next_pulpcore_release: false
```
Comment (daviddavis) [inline template_config.yml:52]: Why is this set to false?

Comment (fao89) [reply template_config.yml:52]: it almost never passes
https://github.com/pulp/pulp_ansible/actions.html?page=1&workflow_file_name=nightly.yml

Comment (daviddavis) [reply template_config.yml:52]: Oh ok. I'm not familiar with the job but I guess we can safely ignore it?

Comment (fao89) [reply template_config.yml:52]: we've been ignoring all this time, I'm also not very familiar with this job

**Review by daviddavis — APPROVED** [review]

**Pull request merged**